common-util/src/main/java/gov/cdc/etldatapipeline/commonutil/json/CustomJsonGeneratorImpl.java

```diff
@@ -16,13 +16,21 @@
 public class CustomJsonGeneratorImpl {
 
     private static final ObjectMapper objectMapper = new ObjectMapper().registerModule(new JavaTimeModule());
+    private static final List<String> DEFAULT_KEYS = Arrays.asList("public_health_case_uid", "act_uid", "observation_uid",
+            "organization_uid", "provider_uid", "patient_uid", "notification_uid");
 
-    public String generateStringJson(Object model) {
+    public String generateStringJson(Object model, String... overrideKeys) {
         try {
             ObjectNode root = objectMapper.createObjectNode();
             ObjectNode schemaNode = root.putObject("schema");
             schemaNode.put("type", "struct");
-            schemaNode.set("fields", generateFieldsArray(model));
+            List<String> pkeys = null;
+            if (overrideKeys != null && overrideKeys.length > 0) {
+                pkeys = Arrays.asList(overrideKeys);
+            } else {
+                pkeys = DEFAULT_KEYS;
+            }
```

Comment:
I'd rather just add `interview_uid` and `investigation_uid` to `DEFAULT_KEYS` without complicating the logic.